src/status_im2/contexts/onboarding/common/carousel/view.cljs

```diff
@@ -0,0 +1,96 @@
+            [status-im2.contexts.onboarding.common.carousel.animation :as animation]))
+
+(def header-text
+  [{:text     (i18n/label :t/join-decentralised-communities)
+    :sub-text (i18n/label :t/participate-in-the-metaverse)}
+   {:text     (i18n/label :t/chat-with-friends)
+    :sub-text (i18n/label :t/with-full-encryption)}
+   {:text     (i18n/label :t/own-your-crypto)
+    :sub-text (i18n/label :t/use-the-multichain-wallet)}
+   {:text     (i18n/label :t/discover-web3)
+    :sub-text (i18n/label :t/explore-the-decentralized-web)}])
+
+(defn header-text-view
+  [index window-width]
+  [rn/view {:style (style/header-text-view window-width)}
+   [quo/text
+    {:style  style/carousel-text
+     :weight :semi-bold
+     :size   :heading-2}
+    (get-in header-text [index :text])]
+   [quo/text
+    {:style style/carousel-sub-text
+     :size  :paragraph-1}
+    (get-in header-text [index :sub-text])]])
+
+(defn content-view
+  [window-width status-bar-height index]
+  (let [content-width (* 4 window-width)]
+    [:<>
+     [rn/image
+      {:style  (style/background-image (* 4 window-width))
+       :source (resources/get-image :onboarding-illustration)}]
+     [rn/view {:style (style/header-container status-bar-height content-width index)}
+      [header-text-view 0 window-width]
+      [header-text-view 1 window-width]
+      [header-text-view 2 window-width]
+      [header-text-view 3 window-width]]]))
+
+(defn progress-bar
+  [static? progress-bar-width]
+  [rn/view
+   {:style {:position       :absolute
+            :top            0
+            :width          progress-bar-width
+            :flex-direction :row}}
+   [rn/view {:style (style/progress-bar-item static? false)}]
+   [rn/view {:style (style/progress-bar-item static? false)}]
+   [rn/view {:style (style/progress-bar-item static? false)}]
+   [rn/view {:style (style/progress-bar-item static? true)}]])
+
+(defn dynamic-progress-bar
+  [progress-bar-width]
+  [:f>
+   (fn []
+     (let [width          (animation/dynamic-progress-bar-width progress-bar-width)
+           original-style style/dynamic-progress-bar
+           animated-style (reanimated/apply-animations-to-style
+                           {:width width}
+                           original-style)]
+       [reanimated/view {:style animated-style}
+        [progress-bar false progress-bar-width]]))])
+
+(defn view
+  []
+  [:f>
+   (fn []
+     (let [window-width       (rf/sub [:dimensions/window-width])
+           view-id            (rf/sub [:view-id])
+           status-bar-height  (:status-bar-height @navigation/constants)
+           progress-bar-width (- window-width 40)
+           carousel-left      (animation/carousel-left-position window-width)
+           original-style     style/carousel-container
+           animated-style     (reanimated/apply-animations-to-style
+                               {:left carousel-left}
+                               original-style)]
```

Comment:
Again I feel unnecessary this pattern of defining a `original-style` and a `animated-style`, we use `original-style` just to apply animations to it, so it would be simpler to apply the animated style on the style definition